test/scripts/cluster.sh

```diff
@@ -2,60 +2,29 @@
 
 ROOT=$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)
 
-function create_kind_cluster {
-  cluster=$1
-  podSubnet=$2
-  svcSubnet=$3
-
-  echo "Creating cluster '$cluster' in podSubnet='$podSubnet' and svcSubnet='$svcSubnet'"
+provision_kind_cluster() {
+  local cluster=$1
 
   kind create cluster --name "$cluster" --config=<<EOF
 apiVersion: kind.x-k8s.io/v1alpha4
 kind: Cluster
-networking:
-  podSubnet: "$podSubnet"
-  serviceSubnet: "$svcSubnet"
 EOF
```

Comment:
Why not just this:
```suggestion
  kind create cluster --name "$cluster"
```

Reply:
True, we don't need this at all. Thanks for spotting it. Fixed in 92db29b